### DISCUSSION TITLE
I would like to know if Microsoft has abandoned MAUI? Is it still worthwhile for me to continue learning and using MAUI

### DISCUSSION THREAD

**Discussion by win4r** [dotnet/maui#19691] [Q&A]
### Description

I have been developing various applications using the MAUI framework since 2023. However, the numerous issues with MAUI have significantly impacted development efficiency and user experience. I would like to know if Microsoft has abandoned MAUI? Is it still worthwhile for me to continue learning and using MAUI?

### Version with bug

8.0.3

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

macOS

**Comment by davidortinau** [comment]
@win4r we continue working hard to resolve issues most impactful to our community. 

Please make sure your specific issues are logged and have all the information to be actionable. I see you opened a Mac WebView issue, thank you!

**Reply by nevotheless** [reply]
aka maintenance mode? 😄
